website/content/docs/agent/options.mdx

```diff
@@ -642,18 +642,33 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
   - `tokens` ((#acl_tokens)) - This object holds all of the configured
     ACL tokens for the agents usage.
 
-    - `master` ((#acl_tokens_master)) - Only used for servers in the [`primary_datacenter`](#primary_datacenter).
+    - `initial_management` ((#acl_tokens_initial_management)) - Only used for
+      servers in the [`primary_datacenter`](#primary_datacenter). This token will
+      be created with management-level permissions if it does not exist. It allows
+      operators to bootstrap the ACL system with a token Secret ID that is well-known.
+
+      The `initial_management` token is only installed when a server acquires cluster
+      leadership. If you would like to install or change it, set the new value for
+      `initial_management` in the configuration for all servers. Once this is done,
+      restart the current leader to force a leader election. If the `initial_management`
+      token is not supplied, then the servers do not create an initial management token.
+      When you provide a value, it should be a UUID. To maintain backwards compatibility
+      and an upgrade path this restriction is not currently enforced but will be in a
+      future major Consul release.
```

Comment:
We should mention which versions this is available in, and what to do if on an older version. Thoughts on whether this should be at the beginning or end of the option description? (We currently do both within this docs page.)

```suggestion
      future major Consul release.
      
      This is available in Consul 1.11+. In prior versions, use [`acl.tokens.master`](#acl_tokens_master).
```

Reply:
Long-term, we need a better solution for handling versions (e.g., versioned docs pages, or a table for the options with a column for version compatibility), but not relevant for this PR.